pkgs/tools/package-management/pdm/default.nix

```
@@ -24,41 +24,35 @@ in
 with python.pkgs;
 buildPythonApplication rec {
   pname = "pdm";
-  version = "1.14.0";
+  version = "2.1.1";
   format = "pyproject";
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "sha256-ZUbcuIRutSoHW5egCpwCKca2IZCgQsRAd72ueDzGySI=";
+    sha256 = "sha256-bffOjgRS6vh7TDD90GwTX2Iol5gWaFOf2+mEE5rUTWo=";
```

```suggestion
    hash = "sha256-bffOjgRS6vh7TDD90GwTX2Iol5gWaFOf2+mEE5rUTWo=";
```

Newfangled hash thingy we prefer using these days.